.pre-commit-config.yaml

```diff
@@ -19,7 +19,7 @@ repos:
     rev: "23.9.1"
     hooks:
       - id: black-jupyter
-        language_version: python3.8
+        language_version: python3.9
```

Comment:
Same comment as for the pypi action

Reply:
For this one I think this will ensure our notebooks comply with 3.9 formatting which I think would be good if people will be running in 3.9 vs 3.8